src/SpacetimeDBClient.cs

```diff
@@ -36,26 +98,13 @@ struct DbOp
             public DbValue? insert;
         }
 
-        /// <summary>
-        /// Called when a connection is established to a spacetimedb instance.
-        /// </summary>
-        public event Action? onConnect;
-
-        /// <summary>
-        /// Called when a connection attempt fails.
-        /// </summary>
-        public event Action<WebSocketError?, string>? onConnectError;
+        public event Action<Identity, string>? onConnect;
```

Comment:
IIUC, these events no longer need to be public, as we configure them via the DbConnectionBuilder instead?

Reply:
Oh this one is tricky; it used to be internal and only used from the builder; but the tests need a way to instantiate a DbConnection without the builder (because its Build() member creates the WS connection)

Reply:
Ok turns out [InternalsVisibleTo] works